steps/reasons-for-appealing/evidence-upload/EvidenceUpload.js

```diff
@@ -109,9 +109,13 @@ class EvidenceUpload extends Question {
 
   values() {
     return {
-      reasonsForAppealing: {
-        otherReasons: this.fields.otherReasonForAppealing.value
-      }
+      evidence:
```

Comment:
I think this should look like this:

```
reasonsForAppealing: {
    evidence: [{blah}]
}
```

As it's within the reasonsForAppealing section

Reply:
Done!